README.md

```diff
@@ -94,9 +122,9 @@ good installation path if you are just testing or doing development on the plugi
 
 * Enable Nexus console: edit `<nexus_dir>/bin/nexus.vmoptions` and change `karaf.startLocalConsole`  to `true`.
 
-  More details here: https://help.sonatype.com/display/NXRM3/Bundle+Development#BundleDevelopment-InstallingBundles
+  More details here: https://help.sonatype.com/display/NXRM3/Bundle+Development+Overview
 
-* Run Nexus' console:
+* Run NXRM' console:
```

Comment:
Trivial: We lost the s here, so gramatically, should be "NXRM's".  Sorry I should have been clear in my request.

Reply:
fixed 9256cd94a7e43a6f43ad6f942b7665f0468ccb7d